scans/zapscan.py

```diff
@@ -0,0 +1,188 @@
+from time import sleep
+import pyotp
+from subprocess import run
+from enum import Enum
+
+from selenium.webdriver import Firefox, FirefoxOptions
+from selenium.webdriver.common.proxy import Proxy
+from selenium.webdriver.common.by import By
+from selenium.common.exceptions import WebDriverException, NoSuchElementException
```

Comment:
I recommend `isort`ing this file for consistency with the rest of the codebase.  (Looks like it already passes `black`!)